src/footer/footer-helper.ts

```diff
@@ -58,13 +60,15 @@ export namespace FooterHelper {
                     alt: "Citizen Collective Common",
                     style: { width: "10rem" },
                 };
-
-            default:
+            case "supportgowhere":
                 return {
-                    src: "https://assets.life.gov.sg/react-design-system/img/logo/lifesg-primary-logo.svg",
-                    alt: "LifeSG",
-                    style: { width: "4rem" },
+                    src: "https://assets.life.gov.sg/react-design-system/img/logo/sgw-logo.png",
+                    alt: "SupportGoWhere",
+                    style: { minWidth: "10rem" },
                 };
+
+            default:
```

Comment:
now that we are not showing the logo by default, we need to show the logo for LifeSG theme